include/h2o/mruby_.h

```diff
@@ -117,12 +119,14 @@ typedef struct st_h2o_mruby_generator_t {
     } refs;
 } h2o_mruby_generator_t;
 
+#define H2O_MRUBY_CALLBACK_ID_NOOP 0
```

Comment:
I am not sure if `0` as a callback ID is allowed. See https://github.com/h2o/h2o/blob/5b71e0a/lib/handler/mruby.c#L773.

Reply:
In current implementation, `0` is actually not meaningful and I believe there're no users who actively uses it because it only raises runtime error which means programming error. OTOH we need a way to exit h2o_mruby_run_fiber